lua/telescope/_extensions/cheat.lua

```diff
@@ -46,7 +46,7 @@ end
 local entry_maker = function(entry)
   return {
     value = entry,
-    ordinal = entry.ns .. " " .. entry.keyword,
+    ordinal = entry.keyword .. ' ' .. entry.ns,
```

Comment:
It better to have ns first so users can write lango = lang go instead of go lang. Category first then item

Reply:
If you say so but your way around i get totally unrelated results. e.g. I typ `lua` and `julia` is like the 3 highest ranking result. With 0 highlighted chars. Just putting that here.
You decide :)
ttyl

Reply:
ohh let me think about and find a work around, or I guess we will keep it like that